doc/changelog.rst

```diff
@@ -3,8 +3,156 @@
+
+Preprocessor
+~~~~~~~~~~~~
+
+-  New preprocessor: `local_solar_time` (:pull:`2258`) by :user:`schlunma`
+-  Read derived variables from other MIP tables (:pull:`2256`) by :user:`bouweandela`
+-  Added special unit conversion m -> DU for total column ozone (toz) (:pull:`2270`) by :user:`schlunma`
+-  Allow cubes as input for `bias` preprocessor (:pull:`2183`) by :user:`schlunma`
+-  Add normalization with statistics to many statistics preprocessors (:pull:`2189`) by :user:`schlunma`
+-  More lazy `extract_levels` preprocessor function (:pull:`2120`) by :user:`bouweandela`
+-  Adding sfcWind derivation from uas and vas  (:pull:`2242`) by :user:`malininae`
+-  Update interval check in resample_hours (:pull:`2362`) by :user:`axel-lauer`
+-  Broadcast properly `cell_measures` when using `extract_shape` with `decomposed: True` (:pull:`2348`) by :user:`sloosvel`
+-  Compute volume from cell_area if available (:pull:`2318`) by :user:`enekomartinmartinez`
+-  Do not expand wildcards for datasets of derived variables where not all input variables are available (:pull:`2374`) by :user:`schlunma`
+-  Cache regridding weights if possible (:pull:`2344`) by :user:`schlunma`
+-  Modernize `regrid_time` and allow setting a common calendar for decadal, yearly, and monthly data (:pull:`2311`) by :user:`schlunma`
+-  Added unstructured linear regridding (:pull:`2350`) by :user:`schlunma`
+-  Add preprocessors `distance_metrics` and `histogram` (:pull:`2299`) by :user:`schlunma`
+
+Release
+~~~~~~~
+
+-  Increase the version number for 2.11.0 release (:pull:`2410`) by :user:`chrisbillowsMO`
+
+Automatic testing
+~~~~~~~~~~~~~~~~~
+
+-  Increase resources for testing installation from conda-forge (:pull:`2297`) by :user:`bouweandela`
+-  Pin pandas to avoid broken `round` function (:pull:`2305`) by :user:`schlunma`
+-  Remove team reviewers from conda lock generation workflow in Github Actions (:pull:`2307`) by :user:`valeriupredoi`
+-  Remove mocking from tests in `tests/unit/preprocessor/_regrid/test_extract_point.py` (:pull:`2193`) by :user:`ehogan`
+-  Pin `pytest-mypy` plugin to `>=0.10.3` comply with new `pytest==8` (:pull:`2315`) by :user:`valeriupredoi`
+-  Fix regridding test for unstructured nearest regridding on OSX (:pull:`2319`) by :user:`schlunma`
+-  Fix flaky regrid test by clearing LRU cache after each test (:pull:`2322`) by :user:`valeriupredoi`
+-  Xfail `tests/integration/cmor/_fixes/test_common.py::test_cl_hybrid_height_coord_fix_metadata` while Iris folk fix behaviour (:pull:`2363`) by :user:`valeriupredoi`
+-  Update codacy reporter orb to latest version (:pull:`2388`) by :user:`valeriupredoi`
+-  Add calls to `conda list` in Github Action test workflows to inspect environment (:pull:`2391`) by :user:`valeriupredoi`
+-  Pin pandas yet again `test_icon` fails again with pandas=2.2.2 (:pull:`2394`) by :user:`valeriupredoi`
+-  Fixed units of cl test data (necessary since iris>=3.8.0) (:pull:`2403`) by :user:`schlunma`
+
+Ux
```

Comment:
I would remove this section and move the pull requests to other sections, as the title Ux does not convey much information for readers of the changelog.

Reply:
For #2326 there is already a second label - enhancement - so we will move it there.

For #2276 this is also seems reasonable to go under enhancement?